screenshots/README.md

```diff
@@ -0,0 +1,171 @@
+
+### [bespin.png](bespin.png)
```

Comment:
What if the header linked directly to the theme file so it could be downloaded directly into the user's theme directory, and then leave the images to link to the PNG file?

Reply:
OK. Let me try...

Reply:
@codebrainz , would the following be sufficient for what you asked for (the linking directly to the theme file for downloading part)?

```
### [bespin theme](../colorschemes.bespin.conf)
```

Reply:
> @codebrainz , would the following be sufficient for what you asked for (the linking directly to the theme file for downloading part)?

Yeah, that's what I meant.